website/docs/docs/cloud/connect-data-platform/connect-redshift-postgresql-alloydb.md

```diff
@@ -63,39 +63,36 @@ chmod 600 ~/.ssh/authorized_keys
-
-`ClientAliveInterval`:<br />
-This value sets a timeout interval in seconds after which if no data has been received from the client, `sshd` will send a message through the encrypted channel to request a response from the client. The default is 0, indicating that these messages will not be sent to the client.<br />
-
-Using default values, tunnels could be terminated prematurely by `sshd`. To solve this problem, the `/etc/ssh/sshd_config` file on the bastion host can be configured with the following values:<br /><br />
-- `ClientAliveCountMax` 10<br />
-- `ClientAliveInterval` 30<br />
-where `ClientAliveCountMax` should be set to a non-zero value and `ClientAliveInterval` should be a value less than the ELB or NLB idle timeout value.<br />
-
-With these settings, unresponsive SSH clients will be disconnected after approximately 300 seconds, helping to prevent tunnel failures.
-</details>
-
 
 ## Configuration
 
-To learn how to optimize performance with data platform-specific configurations in dbt Cloud, refer to [Redshift-specific configuration](/reference/resource-configs/redshift-configs).
+To optimize performance with data platform-specific configurations in dbt Cloud, refer to [Redshift-specific configuration](/reference/resource-configs/redshift-configs).
 
 To grant users or roles database permissions (access rights and privileges), refer to the [Redshift permissions](/reference/database-permissions/redshift-permissions) page or [Postgres permissions](/reference/database-permissions/postgres-permissions) page.
+
+## FAQs
+
+<detailsToggle alt_header="Database Error - could not connect to server: Connection timed out">
+When setting up a database connection using an SSH tunnel, you typically need these parts:
+
+- A Load Balancer (like ELB or NLB) to manage traffic.
+- A bastion host (or jump server) that runs the SSH protocol, acting as a secure entry point.
+- The database itself (such as a Redshift cluster).
+
+dbt Cloud uses an SSH tunnel to connect through the Load Balancer to the database. This tunnel is set up at the start of any dbt job run. If the tunnel drops, the job fails.
+
+Tunnel failures usually happen because:
+
+- The SSH service stops if there's no activity for too long.
+- The Load Balancer cuts off the connection if it's idle.
+- dbt Cloud tries to keep the connection alive by checking in every 30 seconds, and the system will end the connection if there's no response from the SSH service after 300 seconds. This helps avoid drops due to inactivity unless the Load Balancer's timeout is less than 30 seconds.
+
+Bastion hosts might have additional SSH settings to disconnect inactive clients after several checks without a response. By default, it checks three times.
+
+To prevent premature disconnections, you can adjust the settings on the bastion host:
```

Comment:
If these are recommended settings, we should state that, otherwise leave it a little more open ended for customers to configure what's right for them